x-pack/legacy/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -211,6 +211,7 @@ export class HeadlessChromiumDriverFactory {
       mergeMap((err: Error) => Rx.throwError(err))
     );
 
+    // FIXME: log it but don't throwError?
```

Comment:
Added this comment as a way to have a discussion about it: This `throwError` causes some screenshots of dashboards to fail, because of `aborted request` errors. Example:

```
server   error  [12:29:48.632] [error][reporting-tools-plugin] Error: Request to [https://spicy.local/kbn/elasticsearch/_msearch?rest_total_hits_as_int=true&ignore_throttled=true] failed! [net::ERR_ABORTED]
    at MergeMapSubscriber.Rx.fromEvent.pipe.req [as project] (/Users/tsullivan/elastic/kibana/x-pack/legacy/plugins/reporting/server/browsers/chromium/driver_factory/index.ts:220:13)
    at MergeMapSubscriber._tryNext (/Users/tsullivan/elastic/kibana/node_modules/rxjs/src/internal/operators/mergeMap.ts:132:21)
    at MergeMapSubscriber._next (/Users/tsullivan/elastic/kibana/node_modules/rxjs/src/internal/operators/mergeMap.ts:122:12)
    at MergeMapSubscriber.Subscriber.next (/Users/tsullivan/elastic/kibana/node_modules/rxjs/src/internal/Subscriber.ts:102:12)
    at Page.handler (/Users/tsullivan/elastic/kibana/node_modules/rxjs/src/internal/observable/fromEvent.ts:197:20)
    at Page.emit (events.js:189:13)
    at NetworkManager.Page.networkManager.on.event (/Users/tsullivan/elastic/kibana/node_modules/puppeteer-core/lib/Page.js:112:74)
    at NetworkManager.emit (events.js:189:13)
    at NetworkManager._onLoadingFailed (/Users/tsullivan/elastic/kibana/node_modules/puppeteer-core/lib/NetworkManager.js:315:10)
    at CDPSession.emit (events.js:189:13)
    at CDPSession._onMessage (/Users/tsullivan/elastic/kibana/node_modules/puppeteer-core/lib/Connection.js:200:12)
    at Connection._onMessage (/Users/tsullivan/elastic/kibana/node_modules/puppeteer-core/lib/Connection.js:112:17)
    at PipeTransport._dispatch (/Users/tsullivan/elastic/kibana/node_modules/puppeteer-core/lib/PipeTransport.js:61:22)
    at Socket.PipeTransport._eventListeners.helper.addEventListener.buffer (/Users/tsullivan/elastic/kibana/node_modules/puppeteer-core/lib/PipeTransport.js:30:64)
    at Socket.emit (events.js:189:13)
    at addChunk (_stream_readable.js:284:12)
```

This code essentially handles `page.on('requestfailed', ...)` by throwing an error and cancelling out the observable to everything but error handlers. I like the detail we are adding in the logs, but I think logging is maybe the only thing this handler should do.